examples/video-chat/screens/ConnectWithTokenScreen.tsx

```diff
@@ -68,13 +68,13 @@ const ConnectScreen = ({ navigation }: Props) => {
           />
           <TextInput
             onChangeText={onChangeFishjamUrl}
-            value={fishjamUrl}
+            defaultValue={fishjamUrl}
```

Comment:
This is for debugging -> I can easily hardcode room/user name in code so I don't have to type it with each app restart

Reply:
It can be a feature to add for "FIT" property.
Make it that way so in background is this screen as "FILL" and added blur to it. Many ios apps use such patter.